igcollect/unbound.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+#
+# igcollect - Stats for Unbound DNS cache/resolver
+#
+# Copyright (c) 2019 InnoGames GmbH
+#
+
+from __future__ import print_function
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='unbound')
+    return parser.parse_args()
+
+
+def parse_unbound_stats():
+    stats = check_output(
+        ['/usr/local/sbin/unbound-control', 'stats'],
```

Comment:
Is the place of the command fixed there or is it possible that it is somewhere else?